### ISSUE TITLE
Does tflearn.init_graph(seed=1) work? 

### ISSUE THREAD

**Issue by emilwallner** [tflearn/tflearn#865]
There are currently 3 issues regarding this problem:

- https://github.com/tflearn/tflearn/issues/262
- https://github.com/tflearn/tflearn/issues/419
- https://github.com/tflearn/tflearn/issues/757

Is this fixed or is it still a bug? I couldn't make it work, see sample code in issue 757. Does anyone knows how to use tflearn.init_graph(seed=1), or where to start looking if it's a bug. Thanks! 🙌🏽

**Comment by aymericdamien** [comment]
I think that it should be working. But when you run multiple experiments with shuffled data, it will still give different results. Can you retry with 'shuffle=False' in DNN?

**Comment by mpienkosz** [comment]
I have the same problem. Setting random seed with tflearn.init_graph(seed=1) doesn't work, even if I set shuffle to false. I think it is actually a bug in current version of tflearn.

**Comment by emilwallner** [comment]
I also tried the shuffle false, numpy seed, random seed, tflearn seed, and tensorflow seed without success. However, I noticed that is worked on my CPU but not when I tried it on a GPU. I found that the Keras community (also a frontend for Tensorflow) is facing the same problem:

- https://github.com/fchollet/keras/issues/2280
- https://github.com/fchollet/keras/issues/2743
- https://github.com/fchollet/keras/issues/439

Someone mentioned setting the thread to 1 in tensorflow, but that defeats the idea of a using a GPU. I haven't come across any solid solution so far. 

Let me know if anyone works it out.
